src/riak_ql_parser.yrl

```diff
@@ -167,8 +169,8 @@ ColumnDefinition ->
 ColumnConstraint -> not_null : '$1'.
 
 DataType -> datetime : '$1'.
-DataType -> float : '$1'.
-DataType -> int : '$1'.
+DataType -> float_type : '$1'.
```

Comment:
The types are now converted int the parser `int_type => integer` in the parser in commit 6efbe9f042d5bb51c23e11d9273febc15bd17492. I couldn't do it exactly as suggested because the parser expects the data type to to be a two element tuple instead of an atom.